fsspec/caching.py

```diff
@@ -80,6 +80,7 @@ def _makefile(self):
         return mmap.mmap(fd.fileno(), self.size)
 
     def _fetch(self, start, end):
+        logger.debug(f"MMap cache fetching {start}-{end}")
```

Comment:
note that 
```
logger.debug("MMap cache fetching %s-%s", start, end)
```
would be more efficient in that format/%-interpolation would not be even triggered unless logging level at/below logging.DEBUG

Reply:
true, but for rendering integers as strings, I don't think it makes any difference